src/database/models.py

```diff
@@ -797,6 +799,8 @@ def update_visit_log(self, date_time):
                 new = []
                 for day in old.keys():
                     old_format = "%d/%m/%Y"
+                    if len(day) < 10:
```

Comment:
this feels very weird

Reply:
There were a couple cases in the dev DB where the date got entered as a 2 digit year, this cleans that up.